client/src/features/connectedServices/ConnectedServicesPage.tsx

```diff
@@ -0,0 +1,189 @@
+/*!
+ * Copyright 2024 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License
+ */
+
+import { Card, CardBody, CardText, CardTitle } from "reactstrap";
+import cx from "classnames";
+
+import PageLoader from "../../components/PageLoader";
+import { RtkErrorAlert } from "../../components/errors/RtkErrorAlert";
+import connectedServicesApi, {
+  useGetConnectedAccountQuery,
+  useGetConnectionsQuery,
+  useGetProvidersQuery,
+} from "./connectedServices.api";
+import {
+  Connection,
+  ConnectionStatus,
+  Provider,
+} from "./connectedServices.types";
+import { useMemo } from "react";
+import { Loader } from "../../components/Loader";
+import { ExternalLink } from "../../components/ExternalLinks";
+import { BoxArrowUpRight } from "react-bootstrap-icons";
+
+export default function ConnectedServicesPage() {
+  const {
+    data: providers,
+    isLoading: isLoadingProviders,
+    error: providersError,
+  } = useGetProvidersQuery();
+  const { isLoading: isLoadingConnections, error: connectionsError } =
+    useGetConnectionsQuery();
+
+  const isLoading = isLoadingProviders || isLoadingConnections;
+  const error = providersError || connectionsError;
+
+  if (isLoading) {
+    return <PageLoader />;
+  }
+
+  if (error) {
+    return (
+      <div>
+        <RtkErrorAlert error={error} dismissible={false} />
```

Comment:
```suggestion
        <RtkOrNotebooksError error={error} dismissible={false} />
```

That component will handle the error response better
![image](https://github.com/SwissDataScienceCenter/renku-ui/assets/43481553/b04abfe0-cc2d-4471-a21d-f9a83b6fa713)

Reply:
Compared to the current
![image](https://github.com/SwissDataScienceCenter/renku-ui/assets/43481553/c262f9e5-ac2c-4d4b-a74d-93fd387b1a45)